package.json

```
@@ -69,14 +69,14 @@
     "ipfs-utils": "^2.3.0",
     "it-pipe": "^1.0.1",
     "libp2p-utils": "^0.1.0",
+    "libp2p-webrtc-peer": "^9.6.0",
```

This is the only version of the fork.
```suggestion
    "libp2p-webrtc-peer": "^9.7.2",
```